tests/unittests/Foundation/ReferenceFoundation/TestNSURLResponse.mm

```diff
@@ -34,6 +34,7 @@
     NSURLResponse* res =
         [[[NSURLResponse alloc] initWithURL:url MIMEType:@"text/plain" expectedContentLength:0 textEncodingName:nil] autorelease];
     ASSERT_OBJCEQ_MSG(url, res.URL, @"should be the expected url");
+    ASSERT_OBJCEQ(@"path.txt", res.suggestedFilename);
 }
```

Comment:
I'll remove the extra checks, but given how our tests are the name of the test case + extra checking is usual.
We can be strict and keep it to only such that.

---
In reply to: [95277356](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95277356) [](ancestors = 95277356,95277050,95276334)